src/style.css

```diff
@@ -1,3 +1,99 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
+.roboto-medium {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
```

Comment:
Do you need this class?
If not - please remove it. Leaving unused code is a bad practice